services/QuillLMS/spec/support/shared/routing_url_helpers.rb

```diff
@@ -1,4 +1,6 @@
-RSpec.shared_context "routing url helpers" do
+RSpec::ROUTING_URL_HELPERS = "routing url helpers".freeze
```

Comment:
@dandrabik Is this how/where you would define the constant?

Reply:
Yeah, that works.